StoreKitUnitTests/StoreKit2TransactionListenerTests.swift

```diff
@@ -16,25 +16,71 @@ import Nimble
 import StoreKitTest
 import XCTest
 
+@available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
 class StoreKit2TransactionListenerTests: StoreKitConfigTestCase {
 
-    @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+    private var listener: StoreKit2TransactionListener! = nil
+
+    override func setUp() {
+        super.setUp()
+
+        self.listener = .init(delegate: nil)
+    }
+
     func testStopsListeningToTransactions() throws {
         try AvailabilityChecks.iOS15APIAvailableOrSkipTest()
 
-        var listener: StoreKit2TransactionListener? = .init(delegate: nil)
         var handle: Task<Void, Never>?
 
-        expect(listener!.taskHandle).to(beNil())
+        expect(self.listener!.taskHandle).to(beNil())
 
-        listener!.listenForTransactions()
-        handle = listener!.taskHandle
+        self.listener!.listenForTransactions()
+        handle = self.listener!.taskHandle
 
         expect(handle).toNot(beNil())
         expect(handle?.isCancelled) == false
 
-        listener = nil
+        self.listener = nil
         expect(handle?.isCancelled) == true
     }
 
+    // MARK: -
+
+    func testCancelled() async throws {
```

Comment:
nitpick: name tests after the expected output, i.e.: 
`testIsCancelledIsTrueForUserCancelledPurchases`